pytorch_blade/tests/disc/pdl/test_torchscipt_to_mhlo/test_conv_bias.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2021 The BladeDISC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+import torch
+from tests.disc.testing_base import GPUDiscPdlCase
+from tests.quantization import zero_point_dtype
+from torch import nn
+from torch.nn import functional as F
+from torch_blade.utils import torch_version_number
+
+TORCH_VERSION = torch_version_number()
+
+@unittest.skipIf(TORCH_VERSION < (1, 9),
+                 "The patterns corresponding to pytorch before version "
+                 "1.9.0 has not yet been implemented ")
+class TestGPULinear(GPUDiscPdlCase):
+    def test_conv2d(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv2d = nn.Conv2d(16, 33, 3, stride=2)
+
+            def forward(self, x):
+                return self.conv2d(x)
+        model = Model().eval().to(self.device)
+        inp = torch.randn(8, 16, 56, 56).to(self.device)
+        traced_model = torch.jit.trace(model, inp)
+        pdll_files = [
+            os.path.join(self.device_pdll_dir, "conv_bias.pdll")
+        ]
+        pdll_files = ",".join(pdll_files)
+        expect_str = """
+module {
+  func.func @main(%arg0: !torch.nn.Module<"__torch__.___torch_mangle_2.Model"> [unknown], %arg1: tensor<8x16x56x56xf32> [unknown]) -> tensor<?x?x?x?xf32> attributes {tf.entry_function = {input_placements = "cpu,gpu", inputs = "self.1,x", output_placements = "gpu", outputs = "15"}} {
+    %0 = mhlo.constant dense_resource<__elided__> : tensor<33x16x3x3xf32> [unknown]
+    %1 = mhlo.constant dense_resource<__elided__> : tensor<33xf32> [unknown]
+    # CHECK-NOT: torch.aten._convolution
+    # CHECK: mhlo_disc.custom_call_v2
```

Comment:
Add CHECK for the transpose op automaticly inserted by pdll?

Reply:
`transpose` op is inserted in tao_compiler while pdl based pattern match is done before torch2mhlo. Is there any way convenient for us to check the transpose? @wyzero Or maybe current test is enough since transpose related logic has been checked in tao_compiler and we have verified the whole pipeline correctness in e2e test.

Reply:
I think it's ok.